ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/graphql/internal/pojoadapter/TransactionAdapter.java

```diff
@@ -185,4 +185,16 @@ public TransactionAdapter(final TransactionWithMetadata transactionWithMetadata)
     }
     return results;
   }
+
+  public Optional<Boolean> getIsPrivate() {
```

Comment:
does that really have to be an Optional? I think it should just return the boolean ...